vue-frontend/src/components/partials/HeaderV2.vue

```diff
@@ -0,0 +1,231 @@
+import Config from "@/config.js";
+
+export default {
+  data() {
+    return {
+      headerLogoImage: headerLogoImage,
+      careerURL: "/jobs",
+      contactURL: "/contact",
+      blogsURL: Config.BLOG_URL,
+      navbarSticky: false,
+      navbarAnimation: false,
+      navContainerHeight: 133,
+      lastScrollY: 0,
+      activeHomePath: false,
+      activeCareerPath: false,
+      activeContactPath: false,
+    };
+  },
+  components: {},
+  mounted() {
+    let currentRoutePath = this.$router.currentRoute._value.path;
+
+    this.activeHomePath = currentRoutePath == "/";
+
+    this.activeCareerPath = currentRoutePath == this.careerURL;
+
+    this.activeContactPath = currentRoutePath == this.contactURL;
+
+    window.addEventListener("scroll", this.handleScroll);
+    this.navContainerHeight = this.$refs.mainHeader.clientHeight + 30;
+  },
+  unmounted() {
+    window.removeEventListener("scroll", this.handleScroll);
+  },
+  computed: {},
+  methods: {
+    handleScroll() {
+      let wasSticky = this.navbarSticky;
+      this.navbarSticky = window.scrollY > 30;
+
+      // If scroll diff is large, we show navbar with animation
+      let diff = window.scrollY - this.lastScrollY;
+      if (this.navbarSticky != wasSticky && diff > 15) {
+        this.navbarAnimation = true;
+      }
+      if (!this.navbarSticky) {
+        this.navbarAnimation = false;
+      }
+      this.lastScrollY = window.scrollY;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.nav-container {
+  background: #fff;
+  position: relative;
+}
+
+.navbar {
+  padding: 10px 2%;
+  width: 100%;
+  background: #fff;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  z-index: 1;
+  transition: all 0.6s ease-in-out;
+}
+
+.navbar-sticky {
+  position: fixed;
+  left: unset;
+  bottom: unset;
+  box-shadow: 0 13px 35px -12px rgba(35, 35, 35, 0.15);
+}
+
+.navbar-animation {
+  animation: menu_sticky 0.6s ease-in-out;
+}
+
+.btn-link {
+  text-decoration: none;
+}
+
+.v2-button {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.5);
+  border: none;
+}
+
+.v2-black-btn {
+  display: none;
+}
+
+@keyframes menu_sticky {
+  0% {
+    top: -120px;
+    opacity: 0;
+  }
+
+  100% {
+    top: 0;
+    opacity: 1;
+  }
+}
+
+.navbar-nav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.navbar-nav .nav-item-margin .nav-link {
+  margin: 0 20px 0 0;
+  color: #3d3d3d;
+}
+
+.header-logo-image {
+  width: 205px;
+  height: 38.5px;
+}
+
+@include media-breakpoint-up(md) {
+  .navbar-nav .nav-item-margin .nav-link {
+    margin: 0 20px 0 0;
+  }
+
+  .navbar {
+    padding: 20px 0;
+  }
+  .col-flow {
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+}
+@include media-breakpoint-up(lg) {
+  .col-flow {
+    flex-direction: row;
+  }
+  .v2-black-btn {
+    display: block;
+    padding: 10px 30px;
+  }
+  .btn-link {
```

Comment:
This class was also added in app.vue, what is the purpose of that?

Reply:
it create link of black button when view in mobile or tablet view